src/Makefile

```diff
@@ -78,9 +78,9 @@ ifeq (${FC},nvfortran)
     CPP          += cpp -C -P -traditional -Wno-invalid-pp-token -ffreestanding
     ifeq (${DEBUG_L},true)
          OPTS    += -g -O0
-         DM      += -D_B4B
+         # DM      += -D_B4B
```

Comment:
just remove this line

Reply:
I am fine with this change

Reply:
Sorry about the confusion.  I think I have addressed your issue noew.